requests-for-comments/2024-06-21-warehouse-launch-plan.md

```diff
@@ -0,0 +1,51 @@
+## [Marketing] Data warehouse launch plan
+
+The data warehouse has been in an opt-in beta for a while, and has now been opened up to a public open beta. It'll be at least a few weeks before it launches (we need to give current users two weeks notice of pricing changes) BUT that means it's the perfect time to plan. 
+
+### Ahead of launch
+
+- [ ] Publish a Stripe data warehouse tutorial - Marketing
+- [ ] Publish a Hubspot data warehouse tutorial - Marketing
+- [ ] Publish a Postgres data warehouse tutorial - Marketing
+- [ ] Update docs as needed - Marketing
+- [ ] Add Data Warehouse to in-app 'Explore the docs' menu - https://github.com/PostHog/posthog/pull/22715
+- [x] Key art - @lottiecoxon (we have some already, open to new though)
+
+
+### Launch day activity
+
+- [ ] Publish a launch blog piece about why data warehouse is important - 
+- [ ] Update pricing page with data warehouse pricing
+- [ ] Update in-app onboarding with data warehouse option
+- [ ] Share blogpost to HackerNews - Marketing
+- [ ] Email announcement to all users - @joethreepwood
+- [ ] Social push, changelog, Pylon announcement - @joethreepwood
+- [ ] Bookface launch to the YC community - @jamesefhawkins 
+
+### Follow-on activity
+
+- [ ] Source a data warehouse customer story - @joethreepwood
+- [ ] Update onboarding email flow with data warehouse options - @joethreepwood
+
+### Bigger ideas
+
+**Dashboard Templates**
+Is it possible to create dashboard templates that integrate with our integrated sources? E.g. a dashboard for getting Stripe data. 
+
+**Discount Pricing**
+We could offer discounted pricing for the first year to all users who have connected a source during the beta. Easy way to get good will and retain these users - and, if announced early, encourage some early adopters. 
+
+**Data warehouse as a newsletter theme**
```

Comment:
@jtemperton

Reply:
I think something along the lines of "How / Why we decided to build a data warehouse". It would need to get specific, but people enjoy hearing how we decide what to build.

Reply:
Ian is working on something for the website atm: https://github.com/PostHog/posthog.com/issues/8646

Will see how this comes out. Based on the outline, I think we could base a newsletter on elements on Ian's planned article, probably without the more technical bit around how it works.